tests/zenko_tests/node_tests/iam_policies/cloudserver/IAMUser.js

```diff
@@ -0,0 +1,257 @@
+const assert = require('assert');
```

Comment:
Global comment: can we have a linter for these JS files? We can add an extra step in eve as well. This will help us write consistent code.

Reply:
I will take a look into this yeah